Themis/REST/RESTController.swift

```diff
@@ -57,7 +57,13 @@ class RESTController {
             Authentication.shared.authenticated = false
             throw RESTError.unauthorized
         case 403:
-            throw RESTError.forbidden
+            guard let data = data,
+                  let json = try? JSONSerialization.jsonObject(with: data, options: .allowFragments),
+                    let jsonDict = json as? [String: Any], let detail = jsonDict["detail"] as? String else {
+                
+                throw RESTError.forbidden
+            }
+            throw CustomError.error(title: RESTError.forbidden.errorDescription ?? "Test", description: detail)
```

Comment:
Maybe there is a better default description than "Test".

Reply:
Oops that shouldn't be the description, will change it 👍